2025/build-tooling/slides.md

```diff
@@ -91,14 +90,12 @@ graph LR
 - Great developer experience
 - Large and rapidly growing community
 
-
```

Comment:
Not biggest fan of this change, but worth it given that prettier fixes many other things automatically
I can open a pull request with this slidev prettie plugin if it becomes necessary